lms/djangoapps/grades/signals/signals.py

```diff
@@ -33,3 +33,11 @@
                             # made only if the new score is higher than previous.
     ]
 )
+
+# Signal that indicates a course grade needs to be updated
+COURSE_GRADE_UPDATE_REQUESTED = Signal(
+    providing_args=[
+        'user_id',  # Integer User ID
+        'course_id',  # Unicode string representing the course
+    ]
+)
```

Comment:
I'm going to piggyback off of @nasthagiri's PR, https://github.com/edx/edx-platform/pull/13806, and use the `SUBSECTION_SCORE_CHANGED` that she's defined.